src/components/ReduxNetworkProvider.tsx

```diff
@@ -54,11 +62,9 @@ class ReduxNetworkProvider extends React.Component<Props> {
   }
 }
 
-function mapStateToProps(state: { network: NetworkState }) {
-  return {
-    isConnected: state.network.isConnected,
-  };
-}
+const mapStateToProps = (state: AppState, _?: OwnProps) => ({
```

Comment:
this is so that correct type is inferred for the connect component!

I think I struggled fair bit and generally it's much verbose but [this link](https://github.com/piotrwitek/react-redux-typescript-guide#connect-with-react-redux) is what I followed as far as I remember!

Reply:
I've removed it, it still picks the correct defs!